commands/init/mod.ts

```diff
@@ -127,7 +135,7 @@ export const init = command("init", {
     });
 
     if (!flags.json) {
-      yield `✨ Created app "${app.name}"`;
+      yield `✨ Created app "${app.config.name}"`;
       yield "";
       yield fmt.colors.bold("Console URL");
       yield new URL(
```

Comment:
Oh whoops, this URL is wrong now. Should be 
```ts
     yield new URL(
        `/${team}/apps/${app.id}`,
        env.get("PAPERSPACE_CONSOLE_URL"),
      ) + "";
```

Reply:
Also I think the message in general is off because calling this command will trigger a deploy. I wonder if that's actually desirable.

Reply:
I was also a little surprised by that. I wouldn't expect that initializing a cli would _deploy_ something. But if we remove that then is the command really necessary?

It seems like maybe `init` is not the right name, it sounds more like `bootstrap` or something?

Reply:
Ya we need to remove the upsert from the init api and just set up project linking. The command is definitely necessary 😄 It's similar to [`docker init`](https://docs.docker.com/engine/reference/commandline/init/) in that it gets you set up with all of the necessary files. It has the additional step of linking your local directory to a project ID which lets you drop `--projectId` flags from commands.

Reply:
<img width="554" alt="Screen Shot 2023-07-05 at 5 35 10 PM" src="https://github.com/Paperspace/cli/assets/98760247/a3b0b41a-ef9c-4949-b313-1c9856668412">

this is what it looks like